### PR TITLE
core: Add a GenesisConfig for pallet-session

### PR DESCRIPTION
**Pull request by cdamian**
This should address the issue that was brought up here - https://github.com/centrifuge/fudge/pull/63#issuecomment-1591039962

### CODE REVIEW

File: core/src/builder/relay_chain.rs [hunk 1]
```diff
@@ -140,10 +138,10 @@ pub mod types {
 
 	pub struct ParaLifecyclesPrefix;
 	impl StorageInstance for ParaLifecyclesPrefix {
-		const STORAGE_PREFIX: &'static str = "Parachains";
+		const STORAGE_PREFIX: &'static str = "ParaLifecycles";
```
Comment (cdamian) [inline core/src/builder/relay_chain.rs:141]: Funny how we got away with this haha ^^

File: core/src/tests/parachain.rs [hunk 2]
```diff
@@ -179,6 +180,31 @@ fn default_relay_builder(
 				.unwrap(),
 		)
 		.unwrap();
+	state
+		.insert_storage(
+			pallet_session::GenesisConfig::<RRuntime> {
```
Comment (cdamian) [inline core/src/tests/parachain.rs:185]: Providing this genesis config will eventually trigger the `on_new_session` logic for all the relevant pallets such as `session_info` which requires `Some` in `AccountKeys`.

**Comment by cdamian** [comment]
@mustermeiszer PTAL thank you!

**Review by mustermeiszer — APPROVED** [review]
Nice find!

**Pull request merged**